modules/ROOT/pages/managing-instances/instance-details.adoc

```diff
@@ -49,6 +49,10 @@ See xref:managing-instances/custom-endpoints.adoc[Custom endpoints] for more inf
 It runs on container technology and this allows for the AuraDB Instance to allocate dedicated CPU resources.
 
 |Storage
+|Neo4j Aura automates the backup process (you can also create your own on-demand snapshots), storing your data securely in the cloud. Backups are saved in the storage bucket of the cloud provider in the same region as your Neo4j Aura instance.
```

Comment:
```suggestion
|Backups, automated and manual, are stored separately from your instance's allocated storage.
These are saved in the cloud provider's storage bucket in the same region as your Aura instance.
```
Unless this is different now, this information is important. Saying that the backups are stored separately from the instance should clarify enough that they don't encroach on the selected storage space. Also, we try to avoid talking about money, if we can.